Added a PowerShell example snippet

bearmannl opened the pull request.

machish committed:
Update the params of JetDefragment2

Reference: https://github.com/microsoft/Extensible-Storage-Engine/blob/7bdead98c231c4fbb92d75ca8eb6ac1704e3ca3e/dev/ese/src/ese/old.cxx#L3197

drewbatgit committed:
Initial draft of endpoint DLP content

drewbatgit committed:
Small copy edit

drewbatgit committed:
Fixing title

drewbatgit committed:
Checking in the remanding endpoint dlp APIs

drewbatgit committed:
Tech review comments

drewbatgit committed:
Adding Save as category to API list

drewbatgit committed:
Fixing table formatting

drewbatgit committed:
Fixing PR merger errors

TimShererWithAquent committed:
Message fixes: table-syntax-invalid

TimShererWithAquent committed:
Message fixes: misc [BULK UPDATE]

TimShererWithAquent committed:
Additional fix.

bearmannl committed:
added a powershell example snippet

bearmannl committed:
added a powershell example snippet

PRMerger18 commented:
@bearmannl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

drewbatgit committed:
Adding initialization and version functions and associated APIs

drewbatgit committed:
Merge pull request #990 from TimShererWithAquent/us1809622b

Message fixes: misc [BULK UPDATE]

drewbatgit committed:
Adjusting descriptions to get rid of PR merger warnings

Commit:
Fix SRGB renderstates in effect-states.md

Fix missing SRGBWriteEnable render state. TheRealMJP mentioned this issue years ago

https://therealmjp.github.io/posts/correcting-xnas-gamma-correction/

stevewhims committed:
Merge pull request #818 from Brimson/patch-2

Fix SRGB renderstates in effect-states.md

drewbatgit committed:
Merge pull request #989 from TimShererWithAquent/us1809622a

 Message fixes: table-syntax-invalid [BULK UPDATE]

manju-summoner committed:
Update built-in-effects.md

Fix 3D transform effect link

Commit:
Small typo correction

drewbatgit committed:
Adding reference page for DlpMustPasteFromSystemClipboard

drewbatgit committed:
Forgot to check in the new file

stevewhims committed:
Merge pull request #822 from cemdervis/patch-1

Small typo correction

stevewhims committed:
Merge pull request #821 from manju-summoner/patch-1

Fix 3D transform effect link

TimShererWithAquent committed:
Message fixes: alt-text-bad-value and disallowed-domain [BULK UPDATE] (#991)

QuinnRadich committed:
Merging changes synced from https://github.com/MicrosoftDocs/win32 (branch docs)

kokosxD committed:
Move letter inside link text

stevewhims committed:
Edits

stevewhims committed:
Merge pull request #824 from kokosxD/patch-1

Move letter inside link text

drewbatgit committed:
More tech review fixes.

Ohjurot committed:
Fixed position of image and explanation text

The changed text and image was explaining how the example of  "Root Signature Version 1.0" works and looks like, but was placed under "Root Signature Version 1.1"

luki1412 committed:
Invalid return value specified for OpenDatabase (#819)

* Invalid return value specified for OpenDatabase

In the description it says the method returns a Database object but in Return value it says it returns nothing.

Seems like multiple methods of the Installer object have the same issue. Missing return value.

* Edited return value description

* Minor edit

Co-authored-by: McLean Schofield <11544635+mcleanbyron@users.noreply.github.com>

stevewhims committed:
Merge pull request #825 from Ohjurot/ohjurot-d3d12-root-signature-fix

Fixed position of image and explanation text

drewbatgit committed:
Merge pull request #979 from MicrosoftDocs/drewbat/dlp

Drewbat/dlp

stevewhims committed:
updating TOCs and landing pages for restored topics (#992)

drewbatgit committed:
Adding DlpMustPasteFromSystemClipboard to sample header

drewbatgit committed:
Merge pull request #993 from MicrosoftDocs/drewbat/dlp-update

Adding DlpMustPasteFromSystemClipboard to sample header

DJm00n committed:
Cleanup WM_DEVICECHANGE wParam table a bit - sort entries by value

stevewhims committed:
Merge pull request #827 from DJm00n/wm-devicechange-cleanup

Cleanup WM_DEVICECHANGE wParam table a bit - sort entries by value

walbourn committed:
Update xaudio2-redistributable.md (#828)

Added reference to the vcpkg port for this nuget

stevewhims committed:
fixing links to restored topic (#994)

Commit:
Update dx-graphics-hlsl-semantics.md

Separate the input and output semantics to make them easier to read. Had a person mistake `DEPTH[n]` as input semantic

stevewhims committed:
Merge pull request #829 from Brimson/patch-3

Update dx-graphics-hlsl-semantics.md

dubois committed:
Fix blog post link (#826)

Previous link leads to an infinite redirect loop

blueset committed:
Fixing format for RepetitionPattern.Interval (#995)

* Fixing format for RepetitionPattern.Interval

* fix syntax for taskschedulerschema-interval-restarttype-element

* more fixes on the same string

Co-authored-by: Hán Jīn <hanj@microsoft.com>

stevewhims committed:
Stwhi master/dml 1 5 1 (#996)

* new for 1.5

* additional edits

* correction

* moving image

* fixing image link

* a few corrections

* couple of corrections

* correction from PR review

* resolving build errors

MatchaMatch commented:
Hi @bearmannl - it looks like we have a merge conflict here. We can merge this once resolved.

MatchaMatch committed:
Merge pull request #774 from machish/patch-1

Update the params of JetDefragment2

stevewhims committed:
updating for Reunion 0.5 (#997)

bearmannl committed:
Merge branch 'docs' of https://github.com/bearmannl/win32 into docs

bearmannl committed:
merged upstream change

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/win32?pullRequest=817) <br/>All CLA requirements met.

bearmannl commented:
Created a clean fork with the changes, new PR opened: [#830 ](https://github.com/MicrosoftDocs/win32/pull/830).
